server.js

```diff
@@ -201,12 +283,11 @@ app.post('/game/:id/player/:pid/move', validate_gid, validate_pid,
         process_end_game(games[game_id]);
         if (move) {
             augment_move(move, games[game_id]);
-            res.status(200).json(move); // send back move
-            games[game_id].last_move = Date.now();
+            res.status(200).json({game: games[game_id], move: move}); // send back move
```

Comment:
This still modifies the endpoint the photon uses. Can you do something like this:
```
move.game = games[game_id]
res.status(200).json(move);
```